### ISSUE TITLE
Uncaught illegal argument (atom/atom#5366)

### ISSUE THREAD

**Issue by harai** [harai/indent-guide-improved#5]
(Originally from atom/atom#5366 by @glennlopez)

1. Press Tab while writing code
1. type anything ---- error!

**Atom Version:** 0.176.0
**System:** Mac OS X 10.10.2
**Thrown From:** Atom Core
**indent-guide-improved Version:** 1.1.0

# Stack Trace

Uncaught illegal argument

```
At /Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/history.js:124

undefined
```

# Commands

```
     -4:59.3 command-palette:toggle (atom-workspace.workspace.scrollbars-visible-when-scrolling.block-cursor-block.theme-seti-syntax.theme-seti-ui)
  2x -4:54.9 core:move-down (atom-text-editor.editor.mini)
     -4:54.4 core:confirm (atom-text-editor.editor.mini)
     -4:54.4 project-manager:toggle (atom-workspace.workspace.scrollbars-visible-when-scrolling.block-cursor-block.theme-seti-syntax.theme-seti-ui)
     -4:52.5 core:confirm (atom-text-editor.editor.mini)
     -4:47.0 editor:newline (atom-text-editor.editor)
     -4:41.6 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-when-scrolling.block-cursor-block.theme-seti-syntax.theme-seti-ui)
```

# Config

```
{
  "core": {
    "followSymlinks": true,
    "disabledPackages": [
      "filetype-color",
      "git-blame",
      "shadow",
      "gitignore-snippets",
      "git-grep",
      "git-review",
      "atomatigit",
      "git-plus",
      "import",
      "block-selection-mode",
      "auto-indent",
      "atom-beautify",
      "autocomplete-plus-async",
      "linter-scspell",
      "linter-oclitnt",
      "script",
      "red-wavy-underline",
      "time-status",
      "color-gutter",
      "academic",
      "ease-blink",
      "file-icon-supplement",
      "file-icons",
      "whitespace",
      "package-generator",
      "copy-as-rtf",
      "timecop",
      "git-log",
      "carbon-copy",
      "multiple-cursors-clipboard",
      "git-difftool",
      "term2",
      "envygeeks-file-icons",
      "file-type-icons",
      "markdown-writer",
      "markdown-pdf",
      "linter-flake8",
      "overscroll",
      "package-cop",
      "deprecation-cop",
      "tualo-git-context",
      "filesize",
      "git-control",
      "minimap-git-diff",
      "merge-conflicts",
      "autosave",
      "save-session",
      "rulerz"
    ],
    "themes": [
      "seti-ui",
      "seti-syntax"
    ]
  },
  "editor": {
    "fontSize": 12,
    "showIndentGuide": true,
    "autoIndentOnPaste": false,
    "preferredLineLength": 100,
    "tabLength": 3,
    "softTabs": false,
    "softWrapAtPreferredLineLength": true,
    "scrollPastEnd": true,
    "invisibles": {}
  }
}
```

# Installed Packages

```
Sublime-Style-Column-Selection@1.2.3
autocomplete-clang@0.6.1
autocomplete-paths@1.0.1
autocomplete-plus@2.0.6
autocomplete-snippets@1.0.0
block-cursor@0.7.5
git-tab-status@1.5.3
indent-guide-improved@1.1.0
jshint@1.2.0
language-arduino@0.2.1
language-batchfile@0.3.0
language-gitignore@0.1.1
language-ino@0.1.0
language-verilog@0.2.0
language-vhdl@0.6.0
linter@0.11.1
linter-clang@2.22.0
linter-csslint@0.0.11
linter-shellcheck@0.0.6
markdown-format@2.5.0
minimap@3.5.6
minimap-find-and-replace@3.1.0
minimap-highlight-selected@3.1.2
minimap-selection@3.1.1
pdf-view@0.14.0
project-manager@1.15.5
seti-syntax@0.2.1
seti-ui@0.6.3
terminal-status@1.3.5
toolbar@0.0.11
toolbar-main@0.0.5
```
For more details, see atom/atom#5366.

**Comment by harai** [comment]
I've created the same Atom environment on Ubuntu 14.10 but couldn't reproduce the problem.

Atom Editor Version

**Comment by glennlopez** [comment]
@harai for some reason, in some cases the error occurs only when I press tab at the very end of the work I'm coding. I noticed this and found that an easy way to get around it was to press [Enter] at the bottom of my work to make a new line preventing me from typing at the very bottom of the line. Try loading previous code or scripts in the text editor that are actively git tracked. I noticed the errors don't occur on new files.
